spec/acceptance/iis_application_spec.rb

```diff
@@ -85,45 +85,46 @@
   # TestRail ID: C100062
   context 'when setting' do
     describe 'sslflags' do
-      skip("blocked by MODULES-5561")
-      before(:all) do
-        @site_name = SecureRandom.hex(10)
-        @app_name = SecureRandom.hex(10)
-        create_site(@site_name, true)
-        create_path('C:\inetpub\wwwroot')
-        create_path('C:\inetpub\modify')
-        site_hostname = 'www.puppet.local'
-        thumbprint = create_selfsigned_cert(site_hostname)
-        create_app(@site_name, @app_name, 'C:\inetpub\wwwroot')
-        @manifest  = <<-HERE
-          iis_site { '#{@site_name}':
-            ensure          => 'started',
-            physicalpath    => 'C:\\inetpub\\wwwroot',
-            applicationpool => 'DefaultAppPool',
-            bindings        => [
-              {
-                'bindinginformation'   => '*:80:#{site_hostname}',
-                'protocol'             => 'http',
-              },
-              {
-                'bindinginformation'   => '*:443:#{site_hostname}',
-                'protocol'             => 'https',
-                'certificatestorename' => 'MY',
-                'certificatehash'      => '#{thumbprint.downcase}',
-                'sslflags'             => 0,
-              },
-            ],
-          }
-          iis_application { '#{@app_name}':
-            ensure       => 'present',
-            sitename     => '#{@site_name}',
-            physicalpath => 'C:\\inetpub\\modify',
-            sslflags     => ['Ssl','SslRequireCert'],
-          }
-        HERE
-      end
+      skip "MODULES-5561 Test fails for a known reason and has a ticket to fix" do
+        before(:all) do
```

Comment:
Why not just change `describe 'sslflags' do` to `skip 'sslfags - blocked by MODULES-5561' do`?

Then you don't have have to reindent anything.

Reply:
I mean, if you want to be all concise and logical about things, sure...